Makefile

```diff
@@ -54,8 +56,8 @@ check-venv:
 	printf $(VE_MISSING_HELP); \
 	fi
 
-install: venv-create
-	. $(VENV_ACTIVATE_FILE); $(PIP_WRAPPER) install --upgrade hatch==$(HATCH_VERSION) hatchling==$(HATCHLING_VERSION) pip==$(PIP_VERSION) wheel==$(WHEEL_VERSION) pre-commit==$(PRE_COMMIT_VERSION)
+install:
+	. $(VENV_ACTIVATE_FILE); $(PIP_WRAPPER) install --upgrade hatch==$(HATCH_VERSION) hatchling==$(HATCHLING_VERSION) pip==$(PIP_VERSION) wheel==$(WHEEL_VERSION) pre-commit==$(PRE_COMMIT_VERSION) black==$(BLACK_VERSION) isort==$(ISORT_VERSION)
```

Comment:
Shall we update `.pre-commit-config.yaml` so to use `black` and `isort` as installed here instead of having them independently managed by `pre-hook`?

Reply:
We could do that but then why have pre-commit at all? Given our use of pylint in Rally pre-commit does not help much anyway. I'm considering reverting https://github.com/elastic/rally/pull/1546 for this reason.

Reply:
I've not any strong opinion about whether to use pre-commit or not, I just replicated what was done in Rally. Feel free to remove it also from here, if you have any second thought.

I simply don't think is good to use different versions of the tools here and there. Honestly, I think we should just have them as develop dependencies in pyproject.toml (where I expect them to be) and remove them from the Makefile as well.

Reply:
I added a new commit to store the version in a single location. (We should move to pyproject.toml but it's orthogonal)

Reply:
Thanks.